skl2onnx/validate.py

```diff
@@ -0,0 +1,827 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+from time import perf_counter
+from importlib import import_module
+import pickle
+import numpy
+import pandas
+import onnx
+from sklearn import __all__ as sklearn__all__, __version__ as sklearn_version
+from sklearn.base import BaseEstimator
+from sklearn.decomposition import SparseCoder
+from sklearn.ensemble import (
+    VotingClassifier, AdaBoostRegressor,
+)
+try:
+    from sklearn.ensemble import VotingRegressor
+except ImportError:
+    # Available only in 0.21
+    VotingRegressor = 'VotingRegressor'
+from sklearn.feature_selection import SelectFromModel, RFE, RFECV
+from sklearn.linear_model import (
+    LogisticRegression, SGDClassifier, LinearRegression
+)
+from sklearn.model_selection import (
+    train_test_split, GridSearchCV,
+    RandomizedSearchCV
+)
+from sklearn.multiclass import (
+    OneVsRestClassifier, OneVsOneClassifier,
+    OutputCodeClassifier
+)
+from sklearn.multioutput import (
+    MultiOutputRegressor, MultiOutputClassifier,
+)
+try:
+    from sklearn.multioutput import (
+        ClassifierChain, RegressorChain
+    )
+except ImportError:
+    # Available only in 0.21
+    ClassifierChain = 'ClassifierChain'
+    RegressorChain = 'RegressorChain'
+from sklearn.neighbors import LocalOutlierFactor
+from sklearn.svm import SVC, NuSVC
+from sklearn.tree import DecisionTreeRegressor
+from onnxruntime import InferenceSession
+from . import __version__ as ort_version
+from .convert import to_onnx
+from .validate_problems import _problems, find_suitable_problem
+from .helpers.onnx_helper import to_dot
+
+
+def get_opset_number_from_onnx():
+    """
+    Retuns the current *onnx* opset
+    based on the installed version of *onnx*.
+    """
+    return onnx.defs.onnx_opset_version()
+
+
+def sklearn_operators(subfolder=None):
```

Comment:
I guess you could use `sklearn.utils.testing.all_estimators`

Reply:
Fixed.